zio-k8s-client/src/main/scala/com/coralogix/zio/k8s/client/impl/SubresourceClient.scala

```diff
@@ -112,4 +115,160 @@ final class SubresourceClient[T: Encoder: Decoder](
         .response(asJsonAccumulating[T])
         .send(backend)
     }
+
+  override def connect(
+    name: String,
+    namespace: Option[K8sNamespace],
+    customParameters: Map[String, String] = Map.empty
+  ): IO[K8sFailure, AttachedProcessState] = {
+
+    val queueSize = 1024
+    for {
+      nonce     <- ZIO
+                     .effect {
+                       val arr = Array.ofDim[Byte](16)
+                       Random.nextBytes(arr)
+                       new String(Base64.getEncoder.encode(arr))
+                     }
+                     .mapError(toK8sError)
+      stdin     <- ZIO.effect(customParameters.get("stdin").exists(_.toBoolean)).mapError(toK8sError)
```

Comment:
I did not check the schema, just some thoughts:

- If all the `connect` operations have `stdin`, `stdout` and `stderr` then these could be fix built-in parameters, like for example the delete parameters and the watch filters. 
- If it is not available for all, is the `AttachedProcessState` a good result type for those operations?

Reply:
1. They don't, port forward only has a list of ports, and there are port, service and pod proxy endpoints that have action connect, they have just path as option
2. No, it only makes sense for attach and exec